### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pvishalkeerthan**
This PR introduces a complete overhaul of the README for the Forge project. Key updates include:

- A fresh layout with improved spacing and organization.

- Engaging emojis for section headers, enhancing visual appeal.

- Streamlined language for clarity and conciseness.

This new README aims to provide users with a clearer understanding of Forge and its features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,90 @@
...
 
-For Android users, download the APK file from [Snapshot Build](https://downloads.cardforge.org/dailysnapshots/) to your device. 
-On first run, Forge will download all needed data.
+## 🛠️ Installation Guide
 
-## Modes of Play
+### 📥 Desktop Installation
+1. **Latest Releases:** Download the latest version [here](https://github.com/Card-Forge/forge/releases/latest).
+2. **Snapshot Build:** For the latest development version, grab the `forge-gui-desktop` tarball from our [Snapshot Build](https://downloads.cardforge.org/dailysnapshots/).
+   - **Tip:** Extract to a new folder to prevent data loss.
```
Comment (tehdiplomat) [inline README.md:30]: It's not data loss. It's to prevent version conflicts.

File: README.md [hunk 2]
```diff
@@ -1,50 +1,90 @@
...
+3. **User Data Management:** Previous players’ data is preserved during upgrades.
+4. **Java Requirement:** Ensure you have **Java 8 or later** installed.
```
Comment (tehdiplomat) [inline README.md:32]: We should say `17 or later` since that's what it'll be in the near future.

File: README.md [hunk 3]
```diff
@@ -1,50 +1,90 @@
...
+---
+
+## ℹ️ About Forge
 
-Forge has several Quest modes, which is similar but without the overworld map. 
+Forge aims to deliver an immersive and customizable Magic: The Gathering experience for fans around the world. 
 
-You can also play against the AI in a variety of formats, such as Sealed, Draft, Commander and Cube.
+### 📊 Repository Statistics
+- **⭐ Stars:** 969
```
Comment (tehdiplomat) [inline README.md:84]: Is this supposed to be

```
# My Project

![GitHub stars](https://img.shields.io/github/stars/your-username/your-repo?style=social)
![GitHub forks](https://img.shields.io/github/forks/your-username/your-repo?style=social)
![GitHub contributors](https://img.shields.io/github/contributors/your-username/your-repo)

```

Comment (pvishalkeerthan) [reply README.md:84]: I added the repository statistics section for reference. If you think it's unnecessary, I'm happy to remove it

Comment (tehdiplomat) [reply README.md:84]: What I'm saying is you added static values, I believe what I listed above would give you shields that always list the exact number.

Comment (pvishalkeerthan) [reply README.md]: Can you go through the edits I made and check if there are any changes to be made?

**Review by tehdiplomat — APPROVED** [review]

**Pull request merged**